### PR TITLE
Merge all endpoints

### PR DESCRIPTION
**Pull request by tschaffter**
Add `openapi/_internal/nlpsandbox` to merge all NLP Sandbox endpoints.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -197,6 +197,7 @@ jobs:
       matrix:
         api:
         - "tool"
+        - "nlpsandbox"
```
Comment (thomasyu888) [inline .github/workflows/ci.yml:200]: Why is this required?  Can't we just have one base template?  Is it because `Tool` and `ToolDependencies` isn't part of the `datanode`?

**Review by thomasyu888 — APPROVED** [review]
One question, otherwise. LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tschaffter) [reply .github/workflows/ci.yml:200]: For the sake of generating the client, only `nlpsandbox` is required. I'm keeping `tool` around as it may become a specification that we provide in the future to developers who want to cherry pick the endpoint they want to implement. In this use case, they would generate a stub from `tool` and implement the function they are interested in. Does that make sense?
